### DISCUSSION TITLE
Windows version

### DISCUSSION THREAD

**Discussion by plettb** [brminnick/HackerNews#21] [Q&A] [answered]
Ever thought about including a Windows version?  I tried for a few minutes to add it, but I'm too new to Maui to get anywhere.

**Comment by TheCodeTraveler** [comment, marked as answer]
Yup! 

I'm waiting to add Windows support until WinUI is fully supported: https://devblogs.microsoft.com/dotnet/update-on-dotnet-maui/

> To use Windows, you first need to install the Windows App SDK extension for Visual Studio 2022, and then uncomment the TargetFramework node at the top of your csproj file. In a future release, this will be available by default with the extension pre-installed with .NET MAUI.

**Reply by plettb** [reply]
Hmmm...  I'm using VS Version 17.0.0 Preview 7.0 and the WinUI appears to work well.  Still, I haven't exactly exercised it a lot, so I don't know what the issues are.

Of course it's only been out a few days, so I certainly don't mean to pressure you in any way. :)
